src/transformState.js

```diff
@@ -4,8 +4,29 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const prop of actions) {
```

Comment:
[Checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md) 3rd item, use proper names for variables